app/tooling/sales-admin/commands/vault_init.go

```diff
@@ -0,0 +1,96 @@
+package commands
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/ardanlabs/service/foundation/vault"
+)
+
+const credentialsFileName = "/tmp/vault-init.json"
+
+// VaultInit sets up a newly provisioned vault instance
```

Comment:
Said this later. All comments must have proper sentence structure, punctuation, and grammar